db/crud.go

```diff
@@ -1909,6 +1928,24 @@ func (db *Database) updateAndReturnDoc(docid string, allowImport bool, expiry ui
 	// Now that the document has successfully been stored, we can make other db changes:
 	base.DebugfCtx(db.Ctx, base.KeyCRUD, "Stored doc %q / %q as #%v", base.UD(docid), newRevID, doc.Sequence)
 
+	// Delete obsolete attachments from the bucket.
+	if !inConflict {
```

Comment:
I filed CBG-1627 to add support for conflicted documents - can you add a TODO here referencing that?

Reply:
Thank you. Added the comments TODO comments referencing that ticket.